src/network/clientpackethandler.cpp

```diff
@@ -44,10 +45,16 @@ void Client::handleCommand_Hello(NetworkPacket* pkt)
 		return;
 
 	u8 deployed;
-	*pkt >> deployed;
+	u32 auth_mechs;
+	std::string username_legacy; // for case insensitivity
+	*pkt >> deployed >> (u32&) auth_mechs >> username_legacy;
```

Comment:
you can't do `(u32)` for `>>`, therefore I did `(u32&)`. Of course, this doesn't make sense anymore (has been added for debugging), so I'll remove it.